src/modules/hip/kernel/mel_filter_bank.hpp

```diff
@@ -0,0 +1,164 @@
+#include <hip/hip_runtime.h>
+#include "rpp_hip_common.hpp"
+
+__device__ __forceinline__ float calcMel(float *input, int melBin,
+                                         float *weightsDown, int *intervals,
+                                         int fftStride, int fftShift,
+                                         float normFactor)
```

Comment:
Please have each param in seperate line
Also check if we can combine multiple params in single int3/int2 parameter

Reply:
Done